NebulaPatcher/Patches/Dynamic/UIStarmap_Patch.cs

```diff
@@ -0,0 +1,23 @@
+﻿using HarmonyLib;
+using NebulaWorld;
+
+namespace NebulaPatcher.Patches.Dynamic
+{
+    [HarmonyPatch(typeof(UIStarmap))]
+    class UIStarmap_Patch
+    {
+        [HarmonyPostfix]
+        [HarmonyPatch("_OnLateUpdate")]
+        public static void OnLateUpdate_Postfix(UIStarmap __instance)
+        {
+            SimulatedWorld.RenderPlayerNameTagsOnStarmap(__instance);
+        }
+
+        [HarmonyPostfix]
+        [HarmonyPatch("_OnClose")]
+        public static void OnClose_Postfix(UIStarmap __instance)
+        {
+            SimulatedWorld.ClearPlayerNameTagsOnStarmap();
+        }
```

Comment:
Shouldn't this be skipped if not in multiplayer session ?

Reply:
Nothing would happen as there would not be any remote players in the list, but I can add a check